doc/modules/classes.rst

```diff
@@ -902,6 +902,7 @@ details.
    metrics.mean_squared_log_error
    metrics.median_absolute_error
    metrics.r2_score
+   metrics.mean_tweedie_deviance_error
```

Comment:
For the name, maybe `mean_tweedie_deviance` or `mean_deviance_error` could be enough? 

There is also a `d2_score` that is the analogy of `r2_score` that wasn't added in this PR.

Reply:
Maybe `mean_tweedie_error`?
Should we add the `d2_score` to this PR?

Reply:
> Maybe mean_tweedie_error

That could work though it would be a shame to drop the `deviance` term..

> Should we add the d2_score to this PR?

+1 to add it, but in the follow up PR. Reviewers availability and interest is the main concern, and often making smaller PR helps.